src/com/company/Account.java

```diff
@@ -0,0 +1,28 @@
+package com.company;
+
+/**
+ * Created by michaelhilton on 2/18/16.
+ */
+
+public class Account {
+    double principal,rate; int daysActive,accountType;
```

Comment:
Reformat so that fields are easier to read.  Initialize with some values.  Add a comment to describe what they represent.
change fields to be package local.